src/pages/landing-page/subpages/banner/BannerOpenAccess.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Stack, Typography, useTheme } from "@mui/material";
+import {
+  fontSize,
+  fontWeight,
+  gap,
+  margin,
+  padding,
+} from "../../../../styles/constants";
+
+const BannerOpenAccess = () => {
+  const theme = useTheme();
+  return (
+    <Box
+      sx={{
+        marginY: margin.quadruple,
+        width: "100%",
+      }}
+    >
+      <Stack direction="column" justifyContent="center" alignItems="end">
+        <Typography
```

Comment:
Thanks Havier! My concern is most typography styles here are highly coustomized and only used in landing page, so it rarely can be re-used in other components. I agree to add often-used typography in App theme. But for typography in landing page it may cause more code since I need declare in module and interface and then add in theme somehow not so worthy to do that.

Reply:
And if I add them in AppTheme there will too many typos like 'bannerTitleLarge' 'bannerTitleMedium' 'bannerSubtitle' which will be a little bit messy